rosetta-source/src/main/rosetta/base-staticdata-asset-common-type.rosetta

```diff
@@ -134,20 +170,10 @@ type CommodityReferenceFramework: <"Specifies the type of commodity.">
 
 type DigitalAsset extends AssetBase:  <"An Asset that exists only in digital form, eg Bitcoin or Ethereum, that is not backed by other Assets; excludes the digital representation of other Assets, eg coins or Tokenised assets.">
 
-type ListedDerivative extends AssetBase: <"A securitized derivative on another asset.">
-    isExchangeTraded boolean (0..1) <"Identifies whether the derivative is traded on an exchange.">
-    exchange string (0..1) <" Identifies the exchange where the asset is listed, using the scheme definition http://www.fpml.org/coding-scheme/external/exchange-id-MIC-1-0">
-        [metadata scheme]
-    exerciseTerms ExerciseTerms (1..1) <"Specifically the ExerciseDate.">
-    optionType OptionTypeEnum (0..1) <"The type of option, ie Put or Call. Left empty if it is a Future.">
-    strike OptionStrike (0..1) <"Specifies the strike of the option.">
-
-    condition ExchangeTraded:  <"The Exchange must be specified for exchange-traded assets.">
-        if isExchangeTraded
-        then
-            exchange exists
-        else
-            exchange is absent
+type ListedDerivative extends InstrumentBase: <"A securitized derivative on another asset.">
+    expiration string (1..1) <"The Expiration Date of the contract, expressed using the same symbology as the contract identifier.">
```

Comment:
Why `string` and not `date`? I understand that this may get transformed to and from a string when it's serialised into a "message", but it seems odd to have this in the logical model

Reply:
Using a `date` implies that the day is required but listed derivatives are (by convention) identified by their expiry month without the day and often without the year.  I feel a date would be overly constrictive versus a string.

Reply:
I see. In this case, you should expand the description to explain why it's a string and ideally give some examples.

Reply:
found a better attribute elsewhere which defines this:
````
    deliveryTerm string (1..1) <"Also called contract month or delivery month. 
         However, it's not always a month. It is usually expressed using a code, 
         e.g. Z23 would be the Dec 2023 contract, (Z = December). For crude oil, 
         the corresponding contract might be called CLZ23.">
````
This has been added to Phase two PR https://github.com/finos/common-domain-model/pull/3016